MAINTAINERS.md

```diff
@@ -79,5 +79,6 @@ dist plan
 cargo release --no-publish
 
 ```
-[dist]: https://opensource.axo.dev/cargo-dist
+
+[dist]: https://axodotdev.github.io/cargo-dist/
 [the dist book]: https://opensource.axo.dev/cargo-dist/book/workspaces/cargo-release-guide.html#using-cargo-release-with-pull-requests
```

Comment:
I wonder if this book is also available under the url. Might be worth updating too.